test/e2e/mmi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/playwright:v1.40.0-focal AS build
+FROM mcr.microsoft.com/playwright:v1.41.0-focal AS build
```

Comment:
Just curious, was this version bump related to the e2e test fix?

Reply:
It wasn't, but I was getting an error when trying to run the tests locally.

Having said that, it's possible that was just a warning and the error was something else though, as the tests seem to run with the older version in CI.